monstache.go

```diff
@@ -517,7 +518,7 @@ func (config *configOptions) readShards() bool {
 }
 
 func (config *configOptions) dynamicDirectReadList() bool {
-	return len(config.DirectReadNs) == 1 && config.DirectReadNs[0] == ""
+	return len(config.DirectReadNs) == 1 && config.DirectReadNs[0] == "" || config.DirectReadIncludeRegex != ""
```

Comment:
Can we remove this change to how the dynamic direct reads are enabled?  I think this might be a surprising change.  I think we should continue to enable this based strictly on
```toml
direct-read-namespaces = [ "" ]
```

Reply:
I understand what you said is that direct-read-namespaces and direct-read-dynamic-include-regex are mutually exclusive and can only choose one of the two